solr/app.py

```diff
@@ -57,6 +57,8 @@ def json(data, code, headers):
         )
         if code == 200:
             resp.headers['Cache-Control'] = app.config.get('SOLR_CACHE_CONTROL', "public, max-age=600")
+        if 'Set-Cookie' in headers:
```

Comment:
so i'm slightly confused; if we keep this line, do we need the change below in the views?

besides, it only applies to GET requests and not POSTs (not that we allow them :))